src/openmrs-esm-login.tsx

```diff
@@ -3,6 +3,14 @@ import React from "react";
 import ReactDOM from "react-dom";
 import singleSpaReact from "single-spa-react";
 import Root from "./root.component";
+// import { loadConfig } from "@openmrs/esm-root-config";
+import { defineConfigSchema } from "@openmrs/esm-config";
+
+defineConfigSchema("@openmrs/esm-login", {
```

Comment:
👍 I like this -- does it support defining types for configuration values, too?

Reply:
It supports (and does) nothing yet, but that's the idea!
https://docs.google.com/document/d/1Srazq1xfZSmIE7TfyMNKAriPgV2olEMJ68CirYpbgME/edit#
https://pastebin.com/afMRwSvc

Reply:
[Updated draft example usage](https://pastebin.com/KU9NcAp2)

Reply:
The patient search example schema looks good to me.